app/apptestlib.go

```diff
@@ -223,4 +233,61 @@ func (me *TestHelper) TearDown() {
 		StopTestStore()
 		panic(err)
 	}
+	if me.tempWorkspace != "" {
+		os.RemoveAll(me.tempWorkspace)
+	}
+}
+
+type mockPluginSupervisor struct {
+	hooks plugin.Hooks
+}
+
+func (s *mockPluginSupervisor) Start() error {
+	return nil
+}
+
+func (s *mockPluginSupervisor) Stop() error {
+	return nil
+}
+
+func (s *mockPluginSupervisor) Hooks() plugin.Hooks {
+	return s.hooks
+}
+
+func (me *TestHelper) InstallPlugin(manifest *model.Manifest, hooks plugin.Hooks) {
```

Comment:
Wanted to go ahead and build a better way to test plugins. This will install the plugin using hooks passed in via argument (no compiling or writing an executable to disk).